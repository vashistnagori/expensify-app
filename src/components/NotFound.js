import React from "react";
import {Link} from 'react-router-dom';
const NotFound=()=>(
    <div>
    <p>this is 404 page <Link to="/">go home</Link></p>
    </div>
    );
export default NotFound;