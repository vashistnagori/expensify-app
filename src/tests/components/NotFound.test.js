import React from 'react';
import {shallow} from 'enzyme';
import NotFound from '../../components/NotFound';

test("testing not found component",()=>{
    const wrapper=shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});