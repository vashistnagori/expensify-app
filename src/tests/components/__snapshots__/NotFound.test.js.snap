// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing not found component 1`] = `
<div>
  <p>
    this is 404 page 
    <Link
      to="/"
    >
      go home
    </Link>
  </p>
</div>
`;
